homeassistant/components/philips_js/binary_sensor.py

```diff
@@ -0,0 +1,108 @@
+"""Philips TV binary sensors."""
+from __future__ import annotations
+
+from haphilipsjs import PhilipsTV
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import PhilipsTVDataUpdateCoordinator
+from .const import DOMAIN
+
+RECORDINGS_LIST = "recordings"
+RECORDINGS_API_VERSION = 6
+
+
```

Comment:
should be a `@dataclass`, if you had issues with stuff being optional either just give them defaults, or look how others use mixins to solve that.

Reply:
What about using the "const.py"?
In "light.py" or "switch.py" the consts are also just defined as I did.

Reply:
I would have preferred if these constants came from the library. If not, and only used once i think we can just write out the string.

Reply:
Ps. This comment is about missing @dataclass marking.

Reply:
Added dataclass marking and dropped constants.
We can think about adding them to the library as you said.